### ISSUE TITLE
JSON API returns error for "/json/live"

### ISSUE THREAD

**Issue by Cerothen** [Aircoookie/WLED#4205]
### What happened?

When attempting to use the ```http://host/liveview``` page nothing was loading, upon closer inspection the api located at ```http://host/json/live``` was returning the following:

```
{"error": 4}
```

The "Peek" feature in the regular UI works as expected however it is communicating using websocket messages and not with the same JSON api resource

### To Reproduce Bug

Navigate to ```http://host/liveview``` when using version 0.15.0-b6 "Kosen"

![image](https://github.com/user-attachments/assets/3631f105-82d8-409f-98b4-855005716085)

### Expected Behavior

The assumption is that the ```http://host/json/live``` api resource would return a non-error

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.15.0-b6 (build 2410140)

### Which microcontroller/board are you seeing the problem on?

ESP8266

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blazoncek** [comment]
JSON live has been disabled by default in b6.

**Comment by Cerothen** [comment]
Is the intent to have the live view page be removed or switch to use the websocket?

**Comment by blazoncek** [comment]
If you want to use it, custom compile. Otherwise switch to websockects as it is more efficient.

**Comment by netmindz** [comment]
Could we have a check during the build that catches these incompatible set of build options?

**Comment by blazoncek** [comment]
If the JSON live is needed (please get users to upvote this issue) then it can be enabled in release builds.
It was omitted in an attempt to reduce binary size to fit other more important features in the future.
Actual commit: 0bbd6b7c4b677249b584f90abbacde6f44f0ea85

**Comment by Cerothen** [comment]
I'm not directly aware of what's needed to address the websocket however the ```liveview``` page depends on the JSON/live API. I'm just wondering if the liveview page will be rewritten to use websockets or if it is going to be removed

**Comment by blazoncek** [comment]
Web UI (including /liveview) uses websockets.
Nothing special needed.

**Comment by Cerothen** [comment]
the live view page currently does try to access the json/live api.
![image](https://github.com/user-attachments/assets/1162a010-2a9d-4df5-8584-c7b977947f2b)

Perhaps there is a bug?

**Comment by blazoncek** [comment]
No. You have to use websockets in the parent window. UI does that.

**Comment by death2all110** [comment]
Live view is still broken in the release build of 0.15.
Web UI Peek feature works fine.
Opening a new tab/window and visiting /liveview does not work. 
Not sure how I get /liveview to use websocket instead of /json/live

**Comment by blazoncek** [comment]
Please read statements above. It is not broken, it was removed.
If you want to have it for your ESP, you need to compile custom binary.

**Comment by amnesia106** [comment]
How can I adjust the display in the browser live view so that the 16x16 matrix LED panel is shown correctly? In the Peek preview, it is displayed correctly.

**PEEK**
![Image](https://github.com/user-attachments/assets/8cd90fe5-585d-463b-ae4a-1951f1f9845c)
**LIVEVIEW**
![Image](https://github.com/user-attachments/assets/461cc4c0-86b9-4253-b989-3a836173e2db)

**Comment by netmindz** [comment]
If the build only has websocket support then we shouldn't really need the user to add the ?ws modifier

Forgot my ignorance but what does the liveview give that peek doesn't? I.e why use liveview not peek?

**Comment by blazoncek** [comment]
liveview is a frame embedded in UI. it can be called/invoked independently, though.
just a disclaimer: it was not developed to be invoked independent of UI.
